lib/App.server.js

```diff
@@ -29,6 +29,7 @@ App.prototype._init = function(options) {
   this.scriptMapFilename = null;
   this.scriptBaseUrl = (options && options.scriptBaseUrl) || '';
   this.scriptMapBaseUrl = (options && options.scriptMapBaseUrl) || '';
+  this.scriptCrossOrigin = (options && options.scriptCrossOrigin) || '';
```

Comment:
@ericyhwang this should be a boolean, so it should default to `false`. Also, for boolean options I like to them to read like they can only be true/false. Perhaps `this.scriptIsCrossOrigin` or `this.scriptHasCrossOrigin`?